src/js/constants/AppStatus.js

```diff
@@ -9,4 +9,4 @@ var AppStatus = {
   WAITING: 4
 };
 
-module.exports = AppStatus;
+module.exports = Object.freeze(AppStatus);
```

Comment:
Oh yeah, I like this! Shouldn't we do the same in https://github.com/mesosphere/marathon-ui/pull/119 ?